projects/uitdatabank/docs/entry-api/school-events.md

```diff
@@ -0,0 +1,356 @@
+# School events
+
+School events are events that are specifically organized and intented for:
+
+* students (e.g. a school performance)
+* teachers (e.g. an after-school training)
+
+School events are published on [Cultuurkuur](https://www.cultuurkuur.be), but not on UiTinVlaanderen.
+
+> If you’re planning to create school events in UiTdatabank with the Entry API, please contact <content.cultuurkuur@publiq.be> in advance so that we can guide you through this process.
+
+## Types
+
+We distinguish 3 different types of school events:
+
+1. **School performances** are events of which both the date and the location is known in advance.
+2. **Guided tours** are events that have no specific date, but they do have a location.
+3. **Bookable events** are events that have no specific date and location (or the date and location is not known in advance). Both the date and the location are determined in mutual agreement between the organizer or artist and the consumer (school).
+
+### School performances
+
+School performances are events of which both the date and the location is known in advance. For example, a theater performance aimed at a toddler of 3-4 years old in hetpaleis on 14/05/2023, 14:30 - 16:00 PM.
```

Comment:
Maybe it's me but I didn't know what "hetpaleis" was and considered it to be a typo. Maybe make it a link to their website, or put it in quotes (`"hetpaleis"`), or use another example (preferably one that is clearly a name, e.g. starts with a capital letter)?

Reply:
Clarified it with quotes!